doc/source/user_guide/troubleshoot.rst

```diff
@@ -420,7 +422,7 @@ Manually set the location of the executable file
 
 If you have a non-standard install, PyMAPDL might be unable find
 your MAPDL installation. If this is the case, provide the location of MAPDL
-as the first parameter to :func:`launch_mapdl() <ansys.mapdl.core.launch_mapdl>`.
+as the first parameter to :func:`launch_mapdl() <ansys.mapdl.core.launcher.launch_mapdl>`.
 
 **On Windows**
```

Comment:
Earlier, I don't think you bolded the text separating the commands for the operating systems. Again, I recommend always presenting the commands for each OS in the same order.

Reply:
Ok... First Windows, later Linux. Also I'm bolding this kind of text, to be consistent.

Thank you!